package.json

```diff
@@ -1,31 +1,27 @@
 {
-  "name": "highlightjs-supercollider",
-  "version": "1.0.0",
-  "description": "highlight.js syntax definition for the Supercollider language",
-  "main": "supercollider.js",
+  "name": "highlightjs-sclang",
+  "version": "0.1.0",
+  "description": "sclang syntax highlighting plugin for highlight.js",
+  "main": "src/sclang.js",
   "scripts": {
-    "test": "jasmine"
+    "test": "mocha"
   },
-  "files": [
-    "supercollider.js"
-  ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/highlightjs/highlightjs-supercollider.git"
+    "url": "git+https://github.com/highlightjs/highlightjs-cypher.git"
   },
   "keywords": [
+    "sclang",
     "supercollider",
-    "highlight.js",
+    "hljs",
     "highlightjs",
-    "syntax"
+    "highlight.js"
   ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/WinOpsDBA/highlightjs-supercollider/issues"
-  },
-  "homepage": "https://github.com/highlightjs/highlightjs-supercollider#readme",
+  "author": "Dennis Scheiba",
+  "license": "GPL-2.0",
```

Comment:
Shouldn't bugs or homepage just point to GitHub?

Reply:
yes, thx